### PR TITLE
Default GCP service account key for registry storage

### PR DESCRIPTION
**Pull request by ianstanton**
* Part 1/3(?)
* Adds data source for default gcp service account
* Adds service account key resource for default gcp service account
* Some files changed after `terraform fmt`
* Related to astronomer/biz-cloud#72

### CODE REVIEW

File: domain.tf [hunk 1]
```diff
@@ -33,13 +33,13 @@ resource "tls_cert_request" "req" {
...
-    common_name     = "*.${local.base_domain}"
-    organization    = "Astronomer"
+    common_name  = "*.${local.base_domain}"
+    organization = "Astronomer"
   }
 }
 
 resource "acme_certificate" "lets_encrypt" {
-  account_key_pem = acme_registration.user_registration.account_key_pem
+  account_key_pem         = acme_registration.user_registration.account_key_pem
```
Comment (ianstanton) [inline domain.tf]: `terraform fmt` changes

File: firewalls.tf [hunk 2]
```diff
@@ -10,7 +10,7 @@ resource "google_compute_firewall" "bastion_iap_ingress" {
     ports    = ["22"]
   }
 
-  source_ranges = var.iap_cidr_ranges
+  source_ranges           = var.iap_cidr_ranges
```
Comment (ianstanton) [inline firewalls.tf]: `terraform fmt` changes

File: node_pools.tf [hunk 3]
```diff
@@ -13,7 +13,7 @@ resource "google_container_node_pool" "node_pool_mt" {
 
   lifecycle {
     # ignore_changes =["node_config[0].labels", "node_config[0].taint"]
-    ignore_changes =["node_config"]
+    ignore_changes = ["node_config"]
```
Comment (ianstanton) [inline node_pools.tf]: `terraform fmt` changes

File: outputs.tf [hunk 4]
```diff
@@ -21,7 +21,7 @@ output "tls_key" {
 }
 
 output "tls_cert" {
-  value     = <<EOF
+  value = <<EOF
```
Comment (ianstanton) [inline outputs.tf]: `terraform fmt` changes

**Comment by sjmiller609** [comment]
https://github.com/astronomer/terraform-google-astronomer-gcp/releases/tag/0.2.6

**Pull request merged**